charts/internal/gvisor/templates/rbac.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: gvisor
```

Comment:
Can you please harmonize the names? Sometimes you call it `containerd-gvisor`, sometimes only `gvisor`. Please decide for one and use it everywhere.

Reply:
At the moment there is only containerd-gvisor daemon set which handles installation of gvisor with containerd. There can be more and the permissions and the service account can be used for all of them, thats why i left it general with gvisor and not containerd-gvisor.